app-crypt/trousers/files/tcsd.initd

```diff
@@ -1,38 +1,12 @@
 #!/sbin/openrc-run
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2022 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+command=/usr/sbin/tcsd
+description="TrouSerS' TCS daemon (tcsd)"
+command_user="tss:tss"
+
 depend() {
 	use logger
 	need net
 }
-
-checkconfig() {
-	local mod
-	if [ -n "${TPM_MODULES}" ] ; then
-		for mod in ${TPM_MODULES} ; do
-			lsmod | grep -q "^${mod}\b" \
-				|| modprobe ${mod} &>/dev/null \
-				|| ewarn "Failed to load module ${mod}"
-		done
-		# Should we sleep or something to wait for device creation?
-	fi
-	if [ ! -c /dev/tpm ] && [ ! -c /dev/tpm0 ] ; then
-		eerror "No TPM device found!"
-		return 1
```

Comment:
At least this was useful, now it just says "can't start the service" and leaves me guessing what's wrong :(

Reply:
I put that back in